clearpath_motor_drivers/lynx_motor_driver/src/lynx_motor_node.cpp

```diff
@@ -438,6 +438,17 @@ void LynxMotorNode::driverDiagnostic(DiagnosticStatusWrapper & stat, int i)
     }
   }
 
+  // Warning flags
+  for (auto label : WARNING_FLAG_LABELS_) {
+    // isolate and save the bit associated with the flag
+    bool flag = (status_msg_.drivers[i].warning_flags >> label.first) & 0x1;
```

Comment:
In isolation, I assumed `i` was a loop counter, but got confused when I kept expanding up and didn't see a loop.

Not a blocker, but is it worth adding guidelines to our standards to avoid using common loop-counter names as function parameter names?